# Copyright 2007 Sun Microsystems, Inc.  All rights reserved.
# Use is subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
# 

Don't build the sources that require MGAHAL  (matches how we built monolith)

--- src/Makefile.am	2006-11-08 09:16:44.000000000 -0800
+++ src/Makefile.am	2007-01-10 13:11:07.309453000 -0800
@@ -32,7 +32,6 @@
 mga_drv_la_SOURCES = \
          binding.h \
          client.h \
-         clientlx.c \
          mga_arc.c \
          mga_bios.c \
          mga_common.h \
@@ -44,7 +43,6 @@
          mga_esc.c \
          mga_g450pll.c \
          mga.h \
-         mga_halmod.c \
          mga_hwcurs.c \
          mga_macros.h \
          mga_maven.h \
